specs/default/cluster-init/scripts/003_run_reframe.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -x
+set -e
+
+sudo -i
+
+
+INSTALL_DIR=/opt/cycle/nodehealth
+HCHECK_SETTINGS_PATH=$INSTALL_DIR/healthchecks.json
+TEST_PATTERN=$(jq -r '.reframe.pattern' ${HCHECK_SETTINGS_PATH})
+# Determine the OS version
+
+cd $INSTALL_DIR
+
+if ! [[ -f  $INSTALL_DIR/reframe/bin/reframe ]]
+    then
+    mkdir reframe
+    cd reframe
+    curl -L -k https://github.com/JonShelley/reframe/tarball/master  | tar -xz --strip-components 1
+    ./bootstrap.sh
+    ./bin/reframe -V
+    cp azure_nhc/config/azure_centos_7.py azure_nhc/config/azure_centos_7_backup.py
```

Comment:
What we are doing here is cloning jon shelly's repo  and running tests from there. I think we need to clarify if we want to provide an example of how to run the tests or provide actual runnable tests. If former-- we can remove this and provide a simpler hello world example and document it properly and then point the user to jon shelly's repo. The reason this is because people can always use open source reframe: https://reframe-hpc.readthedocs.io/en/stable/index.html.